File: Sources/01XXX/1934/1934.pas
(*
 *  BOJ 1934번 Pascal 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 340 KB / 131,072 KB
 *  소요 시간 : 4 ms / 1,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 *)

VAR T,X,Y,L:LONGINT;
FUNCTION GCD(A,B:LONGINT):LONGINT;
BEGIN
	IF B=0 THEN GCD:=A
	ELSE GCD:=GCD(B,A MOD B);
END;
BEGIN
	READ(T);
	FOR L:=1 TO T DO
	BEGIN
		READ(X,Y);
		WRITELN(X*Y DIV GCD(X,Y));
	END;
END.

File: Sources/01XXX/1934/1934.c
/**
 *  BOJ 1934번 C언어 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 1,116 KB / 131,072 KB
 *  소요 시간 : 0 ms / 1,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

#include <stdio.h>

int gcd(int a,int b){
	int c,max,min;
	max=a>b?a:b;
	min=a>b?b:a;
	while(min){
		c=max%min;
		max=min;
		min=c;
	}
	return max;
}

int main(void){
	int a,b,t,i;
	scanf("%d",&t);
	for(i=0;i<t;i++){
		scanf("%d %d",&a,&b);
		printf("%d\n",a*b/gcd(a,b));
	}
	return 0;
}